src/app/pages/ArticlePage/Byline/index.tsx

```diff
@@ -0,0 +1,253 @@
+    locationBlock,
+  );
+  const image = pathOr(
+    '',
+    ['model', 'blocks', 0, 'model', 'locator'],
+    imageBlock,
+  );
+
+  const contributorBlock = pathOr([], [0], blocks);
+  const authorTopicUrl = pathOr('', ['model', 'topicUrl'], contributorBlock);
+
+  if (!(author && jobRole)) return null;
+
+  const authorTranslated = pathOr('Author', ['byline', 'author'], translations);
+  const articleInformationTranslated = pathOr(
+    'Article information',
+    ['byline', 'articleInformation'],
+    translations,
+  );
+  const jobRoleTranslated = pathOr('Role', ['byline', 'role'], translations);
+
+  const publishedTranslated = pathOr(
+    'Published',
+    ['byline', 'published'],
+    translations,
+  );
+  const reportingFromTranslated = pathOr(
+    'Reporting from',
+    ['byline', 'reportingFrom'],
+    translations,
+  );
+
+  return (
+    <section
+      css={BylineCss.bylineSection}
+      role="region"
+      aria-labelledby="article-byline"
+    >
+      <VisuallyHiddenText id="article-byline" aria-hidden>
+        {articleInformationTranslated}
+      </VisuallyHiddenText>
+      <ul css={BylineCss.bylineList} role="list">
+        {image && (
+          <li
+            css={
+              isRtl
+                ? [BylineCss.imageRtl, BylineCss.Image]
+                : [BylineCss.imageLtr, BylineCss.Image]
+            }
+          >
+            <img css={BylineCss.imageSource} src={image} alt="" />
+          </li>
+        )}
+        <li>
+          {authorTopicUrl ? (
+            <React.Fragment>
+              <VisuallyHiddenText>{`${authorTranslated}, `}</VisuallyHiddenText>
+              <a
+                css={[BylineCss.link, BylineCss.authorLink]}
+                href={authorTopicUrl}
+              >
+                <strong
+                  className="byline__link-text"
+                  css={[
+                    BylineCss.author,
+                    getSansBold(service),
+                    getBodyCopy(script),
+                  ]}
+                >
+                  {author}
+                </strong>
+                {isRtl ? (
+                  <LeftChevron css={BylineCss.authorChevron} />
+                ) : (
+                  <RightChevron css={BylineCss.authorChevron} />
+                )}
+              </a>
+            </React.Fragment>
+          ) : (
+            <span role="text">
+              <VisuallyHiddenText>{`${authorTranslated}, `}</VisuallyHiddenText>
+              <strong
+                css={[
+                  BylineCss.author,
+                  BylineCss.authorLink,
+                  getSansBold(service),
+                  getBodyCopy(script),
+                ]}
+              >
+                {author}
+              </strong>
+            </span>
+          )}
+        </li>
+        <li>
+          <span role="text">
+            <VisuallyHiddenText>{`${jobRoleTranslated}, `} </VisuallyHiddenText>
+            <strong
```

Comment:
Please could we not use strong here? This isn't semantically correct, this isn't important content, please use CSS for font weight of bold. Same goes for the author name above.